apps/maps/migrations/0004_auto_20181107_1309.py

```diff
@@ -14,6 +14,6 @@ class Migration(migrations.Migration):
     operations = [
         migrations.AlterUniqueTogether(
             name='place',
-            unique_together=set([('name', 'country')]),
```

Comment:
This is an auto-generated migration from django code, changing it could effect the detection of changes.

Reply:
I can revert this change, but this would not affect migrations. Literal is much faster since it uses optimized BUILD_MAP and STORE_MAP opcodes rather than generic CALL_FUNCTION.

```bash
> python3.2 -m timeit "d = set('name', 'country')"
1000000 loops, best of 3: 0.975 usec per loop

> python3.2 -m timeit "s = {'name', 'country'}"
1000000 loops, best of 3: 0.409 usec per loop
```